packages/core/src/components/TextField/_Mask.docs.scss

```diff
@@ -10,6 +10,8 @@ Style guide: components.masked-field
 /*
 `<TextField mask={...}>`
 
+A Mask component renders a controlled input field. When the field is blurred, it applies formatting to improve the readability of the value.
+
```

Comment:
Which part is the same? I feel like the main description is talking about what it accomplishes in general: adds "visual and non-visual
cues to a user about the expected value."

While the description under react is talks about how the formatting is applied `onBlur`, which is specific to React. 

How could we combine it?

Reply:
Maybe instead of combining, we could say 
> A Mask component applies formatting to improve the readability of the input value when the field is blurred.

Reply:
What do you think of this? I think the "controlled input field" part is important to include and I moved the "readability" part to the general description

General description:
"A masked field is an enhanced input field that improves readability by providing visual and non-visual cues to a user about the expected value."

React description:
"A Mask component renders a controlled input field that applies formatting to the input value when the field is blurred."

Reply:
@line47 If you are ok with that ^^ I believe I addressed all the feedback